code/__DEFINES/jobs.dm

```diff
@@ -88,3 +91,5 @@
 #define JOB_DISPLAY_ORDER_SECURITY_OFFICER 31
 #define JOB_DISPLAY_ORDER_AI 32

```

Comment:
ditto

Reply:
jobs define already has a number of changes due to reordering display positions, file will already conflict with TG upstream